substrate/client/transaction-pool/tests/integration.rs

```diff
@@ -133,3 +135,155 @@ async fn send_future_and_ready_from_many_accounts_to_relaychain() {
 	assert_eq!(finalized_future, 10_000);
 	assert_eq!(finalized_ready, 10_000);
 }
+
+// Send immportal and mortal txs. Some of the mortal txs are configured to get dropped
+// while others to succeed. Mortal txs are future so not being able to become ready in time and
+// included in blocks result in their dropping.
+#[tokio::test(flavor = "multi_thread")]
+async fn send_future_mortal_txs() {
+	let net = NetworkSpawner::from_toml_with_env_logger(RELAYCHAIN_HIGH_POOL_LIMIT_FATP)
+		.await
+		.unwrap();
+
+	// Wait for the parachain collator to start block production.
+	net.wait_for_block_production("alice").await.unwrap();
+
+	// Create future & ready txs executors.
+	let ws = net.node_rpc_uri("alice").unwrap();
+	let ready_scenario_executor = default_zn_scenario_builder(&net)
+		.with_rpc_uri(ws.clone())
+		.with_start_id(0)
+		.with_nonce_from(Some(0))
+		.with_txs_count(50)
+		.with_executor_id("ready-txs-executor".to_string())
+		.with_remark_recipe(5)
```

Comment:
Would you add some more info on the txs/block distribution?
How many transaction will fit into block with remark 5?

It should help to understand the relation of this param with mortality set below.